backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -162,11 +162,6 @@ trait BackendLifecycleActorFactory {
   def dockerHashCredentials(workflowDescriptor: BackendWorkflowDescriptor,
                             initializationDataOption: Option[BackendInitializationData]
   ): List[Any] = List.empty
-
-  /**
-    * Allows Cromwell to self-identify which cloud it's running on for runtime attribute purposes
-    */
-  def platform: Option[Platform] = None
```

Comment:
Since this actor factory isnt platform specific i left this out. There is a new trait to specify platform